cmd/ci-operator/main.go

```diff
@@ -348,7 +345,7 @@ func bindOptions(flag *flag.FlagSet) *options {
 	flag.StringVar(&opt.branch, "branch", "", "Branch of the project (used by configresolver)")
 	flag.StringVar(&opt.variant, "variant", "", "Variant of the project's ci-operator config (used by configresolver)")
 
-	flag.StringVar(&opt.kubeconfig, "kubeconfig", "", "Path to .kube/config file. First config whose host matches the cluster is used to access imagestreamtags. If not set or no matching config , use the anonymous user")
+	flag.String("kubeconfig", "", "Legecay flag kept for compatibility reasons. Doesn't do anything.")
 
 	flag.StringVar(&opt.pullSecretPath, "image-import-pull-secret", "", "A set of dockercfg credentials used to import images for the tag_specification.")
```

Comment:
Naively I thought THIS was the flag we needed to remove? WTF?

Reply:
Not sure that that thing does, I can check in a follow-up if its still needed. I think Im going to remove the functionality bit by bit